Subject: fix compile error due to API change
From: Erwin Rol <erwin@erwinrol.com>

The new Xserver has a changed API, this patch adds an ifdef to be able
to compile the driver for old and new Xservers.

Signed-off-by:  Erwin Rol <erwin@erwinrol.com>

--- xf86-input-tslib-0.0.4/src/tslib.c.orig	2009-06-11 18:38:15.000000000 +0200
+++ xf86-input-tslib-0.0.4/src/tslib.c	2009-06-11 18:49:25.000000000 +0200
@@ -218,7 +218,9 @@
 
 		if (InitValuatorClassDeviceStruct(device,
 						  2,
+#if GET_ABI_MAJOR(ABI_XINPUT_VERSION) < 3
 						  xf86GetMotionEvents,
+#endif
 						  0, Absolute) == FALSE) {
 			ErrorF("unable to allocate Valuator class device\n");
 			return !Success;
